packages/scaffold/src/views/w3m-register-account-name-view/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { css } from 'lit'
+
+export default css`
+  wui-flex {
+    width: 100%;
+  }
+
+  .suggestion {
+    background: var(--wui-gray-glass-002);
+    border-radius: var(--wui-border-radius-xs);
+  }
+
+  .suggestion:hover {
+    background-color: var(--wui-gray-glass-005);
+    cursor: pointer;
+  }
+
+  .suggested-name {
+    max-width: 220px;
```

Comment:
Is this value also applicable for `mobile` or smaller screens? Is it possible to use relative units?